Makefile

```diff
@@ -1,5 +1,8 @@
 include .env
 
+check-configs: 
```

Comment:
This won't run without it's cmd line args, and Make doesn't allow them... Still need a permanent solution here

Reply:
takes in makefile input arguments now